medical-clinic/src/main/resources/liquibase-changelog/versions/0003_MEDICAL_CLINIC.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3
+      author: Mateusz
+      changes:
+        - createTable:
+            tableName: APPOINTMENT
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: appointment_id
+              - column:
+                  name: doctor_id
+                  type: int
+                  constraints:
+                    foreignKey: true
+                    foreignKeyName: id
+              - column:
+                  name: patient_id
+                  type: int
+                  constraints:
+                    foreignKey: true
+                    foreignKeyName: id
```

Comment:
nazwij inaczej to foreign key -> prosilem was aby foreign key zaczynaly sie od fk_ primary key od pk_ a unique alob uq_ albo unique